jetty-http3/http3-tests/pom.xml

```diff
@@ -10,6 +10,11 @@
   <artifactId>http3-tests</artifactId>
   <name>Jetty :: HTTP3 :: Tests</name>
 
+  <properties>
```

Comment:
@sbordet I assume we should not deploy those tests as part of the release and was a mistake? or I am wrong?

Reply:
This module will only produce a test jar (no sources), so it won't be deployed anyway, right?
Ah, I see it deployed an empty jar 😒 
So yes let's skip deploy for this module.

Reply:
after digging a bit the jar is not empty as it included some content we put in `target/classes` as part of the build such: generated MANIFEST.MF, LICENSE and NOTICE.txt